docs/user/build/install/jdk.rst

```diff
@@ -105,20 +112,17 @@ One technique is to set up a batch file similar to the following:
    
    Placing JAVA_HOME on the path before System32 shortcuts this annoying "feature".
 
-Java Extensions (Optional)
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+Java Extensions (Pre Java 9)
```

Comment:
Everywhere else references to Java 9 have been removed and replaced with Java 11.
Perhaps use "Java 8 and before" instead, for consistency?